### PR TITLE
feat: added support for youtube chapters

### LINKED ISSUE #39: Add chapters pulled from youtube
If a video or podcast has chapters in the description, e.g. https://www.youtube.com/watch?v=sNLbg9BUV4Y

Create breaks and add them as headers in the transcript.

Might be some clues on how to gather from youtube here: https://stackoverflow.com/questions/63821605/how-do-i-get-info-about-a-youtube-videos-chapters-from-the-api

### PR DESCRIPTION
**Pull request by masterchief164**
This PR fixes #39.

### CODE REVIEW

File: app/application.py [hunk 1]
```diff
@@ -8,24 +8,97 @@
...
+                     f'{current_chapter[1]}\n'
+                     f'CHAPTER{current_chapter[0]}NAME='
+                     f'{current_chapter[2]}\n')
+
+
+def split_mp4(chapters: list, download_filename: str, download_name: str) -> None:
+    # current_duration = subprocess.run(['ffprobe', '-i', download_filename,
+    # '-show_entries', 'format=duration',
+    # '-v', 'quiet', '-of', 'csv="p=0"'],
+    # capture_output=True, encoding='UTF8')
```
Comment (adamjonas) [inline app/application.py:75]: rm comments

File: setup.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 setup(
     name='tstbtc',
     version='1.0.0',
-    author='Peter Tyonum',
-    author_email='withtvpeter@gmail.com',
+    author='Peter Tyonum, Shaswat Gupta',
+    author_email='withtvpeter@gmail.com, shaswat2001.sg@gmail.com',
     license='MIT',
```
Comment (adamjonas) [inline setup.py:13]: should be a separate PR or at least separate commit

File: app/application.py [hunk 3]
```diff
@@ -8,24 +8,97 @@
...
+        name = video.title
+        with open(name + '.description', 'w') as f:
+            f.write(video.description)
         stream = video.streams.get_by_itag(18)
         stream.download()
         return stream.default_filename
     except Exception as e:
         print("Error downloading video")
+        if name:
+            os.remove(name + '.description')
```
Comment (adamjonas) [inline app/application.py:31]: having issues here for 
```
File "/Users/jonas/chaincode/yt2btc/venv/lib/python3.9/site-packages/app/application.py", line 31, in download_video
    os.remove(name + '.description')
FileNotFoundError: [Errno 2] No such file or directory: "Mark 'Murch' Erhardt and UTXO Management/Enterprise Wallets - Episode 8 of the Chaincode Podcast.description
```

**Comment by adamjonas** [comment]
I left some comments on https://github.com/adamjonas/bitcointranscripts/pull/25/files but this is really cool!

Getting those timestamps right is going to be a challenge but it's awesome to see them.

File: transcriber.py [hunk 4]
```diff
@@ -87,15 +87,36 @@ def add(
                 print("Supplied date is invalid")
                 return
         for video in videos:
+            result = ""
             print("Transcribing video: " + video)
             filename = application.download_video(video)
+            print()
```
[the remark this answers is not in the document]
Comment (masterchief164) [reply transcriber.py:93]: This is not a duplicate line. It's needed to first go to the next line and then add a new line in the CLI to mark separation between the items.

**Comment by adamjonas** [comment]
I'm going to merge but opening #53 as a follow up/
